<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VaC_rebootClass</class>
 <widget class="QMainWindow" name="VaC_rebootClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VaC_reboot</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_UserName">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>6</y>
      <width>132</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Serial">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>30</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>6</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>序列号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>重启验证</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VaC_reboot.qrc"/>
 </resources>
 <connections/>
</ui>
